test/features/step_definitions/sdk_steps.rb

```diff
@@ -41,22 +46,43 @@
 
 Then(/^I should reach the complete step$/) do
   steps %Q{
-    Then page_title () should contain "Verification complete"
-    Then I should not see "back"
+    Then page_title () should contain "#{i18n.translate('complete.message')}"
+    Then page should not have back button
   }
 end
 
-Then(/^I can navigate back to the previous page with title "([^"]*)"$/) do | title |
+Then(/^I can navigate back to the previous page with title "([^"]*)"$/) do | key |
+  title = i18n.translate(key)
   steps %Q{
     When I click on back ()
     Then page_title () should contain "#{title}"
   }
 end
 
 When(/^I upload my document and selfie$/) do
+  selfie = i18n.translate('capture.face.upload_title')
   steps %Q{
     When I try to upload passport
-    Then page_title () should contain "Selfie"
+    Then page_title () should contain "#{selfie}"
     When I try to upload one_face
   }
 end
+
+Then(/^(.*) should include "([^"]*)"$/) do | page_element, key|
+  text = i18n.translate(key)
+  steps %Q{
+    Then #{page_element} () should contain "#{text}"
+  }
+end
+
+Then(/page should not have back button$/) do
+  #This won't throw an exeption and will save time
+  @driver.find_elements(:css, '.onfido-sdk-ui-NavigationBar-back').size() == 0
```

Comment:
Try to extract `@driver.find_elements(:css, '.onfido-sdk-ui-NavigationBar-back')` to SDK class in `sdk_steps.rb` so that we don't use the css class reference in two places.
Also make assertion using expect, for current implementation it's: `expect(@driver.find_elements(:css, '.onfido-sdk-ui-NavigationBar-back').size()).to eq 0